src/Components/Blazor/Templates/src/Microsoft.AspNetCore.Blazor.Templates.csproj

```diff
@@ -49,8 +61,4 @@
       <Output TaskParameter="OutputPath" ItemName="Content" />
     </GenerateFileFromTemplate>
   </Target>
-  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
-  <!-- Override SDK targets. This project does not produce a .dll. -->
-  <Target Name="CoreCompile" />
-  <Target Name="GetTargetPath" />
```

Comment:
Unfortunately I can't find any way to make a project fully incremental when it is both:
- a csproj
- does not produce a dll

I spent a while debugging this and the simplest solution is to produce a dll. We have a nuspec already so it doesn't get shipped anywhere.